api/src/test/scala/hmda/api/http/LarHttpApiSpec.scala

```diff
@@ -72,26 +75,42 @@ class LarHttpApiSpec extends WordSpec with MustMatchers with ScalatestRouteTest
     "filter syntactical, validity, or quality only for invalid LAR with all 3 kinds of errors" in {
       val badLoanType = lar.loan.copy(loanType = 0, amount = 900, propertyType = 2)
       val badLar = lar.copy(agencyCode = 0, loan = badLoanType, purchaserType = 4)
-      Post("/lar/validate", badLar) ~> larRoutes ~> check {
+      postWithCfpbHeaders("/lar/validate", badLar) ~> larRoutes ~> check {
         status mustEqual StatusCodes.OK
         responseAs[List[ValidationError]].length mustBe 3
       }
       //should fail S020
-      Post("/lar/validate?check=syntactical", badLar) ~> larRoutes ~> check {
+      postWithCfpbHeaders("/lar/validate?check=syntactical", badLar) ~> larRoutes ~> check {
         status mustEqual StatusCodes.OK
         responseAs[List[ValidationError]].length mustBe 1
       }
       //should fail V220
-      Post("/lar/validate?check=validity", badLar) ~> larRoutes ~> check {
+      postWithCfpbHeaders("/lar/validate?check=validity", badLar) ~> larRoutes ~> check {
         status mustEqual StatusCodes.OK
         responseAs[List[ValidationError]].length mustBe 1
       }
       //should fail Q036
-      Post("/lar/validate?check=quality", badLar) ~> larRoutes ~> check {
+      postWithCfpbHeaders("/lar/validate?check=quality", badLar) ~> larRoutes ~> check {
         status mustEqual StatusCodes.OK
         responseAs[List[ValidationError]].length mustBe 1
       }
     }
   }
 
+  "reject requests without 'CFPB-HMDA-Username' header" in {
```

Comment:
Small thing here, but the Username test is adding the `institutionsHeader` (and vice versa).